WiserTaskScheduler/WiserTaskScheduler/Modules/Branches/Services/BranchQueueService.cs

```diff
@@ -1513,6 +1524,7 @@ await using (var environmentCommand = branchConnection.CreateCommand())
             }
             catch (Exception exception)
             {
+                logger.LogError(exception, $"Failed to merge the branch '{settings.DatabaseName}'.");
```

Comment:
Please use the logService for logging. This ensures the log is written to the database and/or a Slack channel.

Reply:
Done